kolibri/locale/hi_IN/LC_FRONTEND_MESSAGES/management_module-messages.json

```diff
@@ -2,102 +2,102 @@
-  "classnameEditModal.duplicateName":"उस नाम के साथ एक वर्ग पहले से मौजूद है",
-  "classEnrollPage.enrollUsers":"उपयोगकर्ताओं को पंजीकृत करें",
-  "classEnrollPage.tableTitle":"इस वर्ग में शिक्षार्थियों को प्रबंधित करें",
-  "classEnrollPage.searchText":"एक नौसिखिया या कोच मिल...",
+  "classnameEditModal.update":"Update (अपडेट करें)",
+  "classnameEditModal.duplicateName":"इस नाम के साथ एक वर्ग पहले से मौजूद है",
+  "classEnrollPage.enrollUsers":"उपयोगकर्ताओं का दाखिला करें",
+  "classEnrollPage.tableTitle":"इस वर्ग के शिक्षार्थियों को प्रबंधित करें",
+  "classEnrollPage.searchText":"शिक्षार्थी या कोच को ढूंढे",
   "classEnrollPage.users":"उपयोगकर्ता",
   "classEnrollPage.fullName":"पूरा नाम",
   "classEnrollPage.username":"उपयोगकर्ता नाम",
-  "classEnrollPage.role":"भूमिका",
+  "classEnrollPage.role":"Role (भूमिका)",
   "classEnrollPage.learner":"सिखने वाला",
   "classEnrollPage.coach":"प्रशिक्षक",
-  "classEnrollPage.remove":"हटाएं",
-  "classEnrollPage.noUsersExist":"इस वर्ग में कोई उपयोगकर्ता",
-  "classEnrollPage.allUsersFilteredOut":"कोई मेल खाने वाला उपयोगकर्ता",
+  "classEnrollPage.remove":"हटाएं (रीमूव)",
+  "classEnrollPage.noUsersExist":"इस वर्ग में कोई उपयोगकर्ता(users) नहीं है",
+  "classEnrollPage.allUsersFilteredOut":"कोई मेल खाने वाला उपयोगकर्ता (users) नहीं है",
   "roleSwitcher.admin":"व्यवस्थापक",
   "roleSwitcher.coach":"प्रशिक्षक",
   "roleSwitcher.learner":"सिखने वाला",
   "userRemoveModal.modalTitle":"वर्ग से उपयोगकर्ता को निकालें",
   "userRemoveModal.remove":"वर्ग से निकालें",
   "userRemoveModal.cancel":"रद्द करें",
-  "userRemoveModal.deleteConfirmation":"क्या आप { username } से { classname } निकालने के लिए सुनिश्चित हैं?",
-  "userRemoveModal.accessReassurance":"आप अभी भी { sectionTabName } से इस खाते तक पहुँच सकते हैं",
+  "userRemoveModal.deleteConfirmation":"क्या आप वाकई { username } को { classname } निकालना चाहते है?",
+  "userRemoveModal.accessReassurance":"आप अभी भी { sectionTabName } से इस खाते (account) तक पहुँच सकते हैं",
   "userRemoveModal.usersTab":"उपयोगकर्ता",
-  "confirmEnrollmentModal.confirmEnrollment":"चयनित उपयोगकर्ताओं के नामांकन की पुष्टि करें",
-  "confirmEnrollmentModal.areYouSure":"आप सुनिश्चित करें कि आप निम्न उपयोगकर्ताओं में भर्ती करना चाहते हैं",
-  "confirmEnrollmentModal.noGoBack":"नहीं, वापस जाना",
-  "confirmEnrollmentModal.yesEnrollUsers":"हाँ, उपयोगकर्ताओं को पंजीकृत करें",
-  "managementClassEnroll.backToClassDetails":"श्रेणी विवरण के लिए वापस",
-  "managementClassEnroll.enrollSelectedUsers":"समीक्षा & सहेजें",
-  "managementClassEnroll.selectLearners":"में नामांकन के लिए उपयोगकर्ताओं का चयन करें",
-  "managementClassEnroll.showingAllUnassigned":"सभी उपयोगकर्ता वर्तमान में नहीं दिखा रहा है इस कक्षा में दाखिला",
-  "managementClassEnroll.searchForUser":"एक उपयोगकर्ता के लिए खोज",
+  "confirmEnrollmentModal.confirmEnrollment":"चयनित उपयोगकर्ताओं के दाखिले की पुष्टि करें",
+  "confirmEnrollmentModal.areYouSure":"आप वाकई निम्न उपयोगकर्ताओं को ______ में भर्ती करना चाहते हैं",
```

Comment:
is this intentional?

Reply:
It's what the translator wrote.

Reply:
My guess is that the English string: `Are you sure you want to enroll the following users into` does not work nicely in Hindi.

This string itself seems to be problematic, because we should be inserting the class name into the string with a variable.

Reply:
oh, damn.

```
<p>{{ $tr('areYouSure') }} <strong>{{ className }}</strong>?</p>
```

Reply:
i'll make an issue